src/main/java/org/opentripplanner/routing/core/TraverseModeSet.java

```diff
@@ -42,8 +42,12 @@
 
     private static final int MODE_AIRPLANE = 4096;
 
+    private static final int MODE_TROLLEYBUS = 8192;
+
+    private static final int MODE_MONORAIL = 16384;
```

Comment:
Now that we have so many of these it might be better to define them by bit-shifting 1 by successive integers, to avoid any errors in writing out the powers of 2 as integer literals.